src/fastoad/notebooks/02_explanation/02_pure_python.ipynb

```diff
@@ -0,0 +1,850 @@
+{
```

Comment:
<p><code>    # Let's start by converting the quantities in imperial units </code></p><p><code>    mtow_lbs = mtow * 2.20462 </code></p><p><code>    wing_area_ft2 = wing_area * 10.7639 </code></p><p>Please see my comment in the file itself and update here if needed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Changed on file and on notebook in 4d319923